src/Result/ServerSyncStatusNodeInformation.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Result;
+
+class ServerSyncStatusNodeInformation extends AbstractResult
+{
+    public function getHeaders(): int
+    {
+        return $this->getData()['headers'];
+    }
+
+    public function getBlocks(): int
+    {
+        return $this->getData()['blocks'];
+    }
+
+    public function getVerificationProgress(): float
```

Comment:
Not sure if we should return a string or our PreciseNumber here, float might get some rounding issues but probably is fine if the values only have a few decimals. in that case we can keep float I think.

Reply:
it's quite a bit, i'll change to PreciseNumber

`float(0.9999998220006474)`